### ISSUE TITLE
just not open

### ISSUE THREAD

**Issue by ncuxomozg** [avocode/avocode#700]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.5.2
Source UID: 147377
```
Error: invalid block type
  at Zlib._handle.onerror (zlib.js:345:17)

```

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.

**Comment by ncuxomozg** [comment]
Slightly different error now. Endless processing message. I was placed here picture with my problem. 
Version 2.5.3. 
http://pasteboard.co/1lTJYZNa.png

**Comment by petrbrzek** [comment]
Thanks for the feedback. Did any exception window popup or the error was silent?

**Comment by ncuxomozg** [comment]
Only "processing failed" message and the red background.

**Comment by petrbrzek** [comment]
Hi, we've released 2.5.4 that should fix this issue. If you still have this problem in the new app, feel free to report it as a new issue.
